scripts/package/package_android_sdk.sh

```diff
@@ -23,9 +23,9 @@
 
 set -e
 
-ANDROID_PLATFORM=29
-ANDROID_TARGET_API_LEVEL=29
-ANDROID_BUILD_TOOLS_VERSION=29.0.3
+ANDROID_PLATFORM=30
```

Comment:
Also check:

https://github.com/defold/defold/blob/dev/build_tools/waf_dynamo.py#L62-L66
https://github.com/defold/defold/blob/dev/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properti
es#L444
https://github.com/defold/defold/blob/dev/editor/resources/meta.edn#L605

Reply:
Also https://github.com/defold/defold/blob/dev/scripts/build.py#L618

Reply:
I have now packaged the version 30 for darwin/linux and uploaded it to S3, so the next CI build could work (after updating the scripts mentioned above)